lib/arrow.cpp

```diff
@@ -33,6 +33,11 @@
 #include <sstream>
 
 namespace grnarrow {
+  enum arrow_type_mode {
```

Comment:
I used `enum` (not `enum class`) only because I wanted to use briefly bit flags.
If we do not need use bit flags, I will change it into `enum class`.
(Related: https://github.com/groonga/groonga/pull/1263#discussion_r707374227)

> The current values just show whether dictionary is used or not but `arrow_type` is too generic.
Okey, I will fix it.